charts/catalog-server/README.md

```diff
@@ -52,6 +52,7 @@ A Helm chart for RADAR-base catalogue server. This application creates RADAR-bas
 | kafka_num_brokers | int | `3` | number of Kafka brokers to look for |
 | kafka | string | `"cp-kafka-headless:9092"` | URI of Kafka brokers |
 | schema_registry | string | `"http://cp-schema-registry:8081"` | URL of the confluent schema registry |
+| kafkaProperties."security.protocol" | string | `"PLAINTEXT"` | Protocol used to communicate with brokers. Valid values are: PLAINTEXT, SSL, SASL_PLAINTEXT, SASL_SSL. |
```

Comment:
keys with dots in them are a pain to reference in go templates. I'm wondering in this case whether this can be a list of name/values instead. The only downside is that you can't really override defaults, so the default should just be `[]`. Alternatively, the `_` symbol is not used in actual kafka properties, so in the template you could do

```go
{{ $key | replace "_" "." }}
```
and in the comments document that this replacement is taking place.